__tests__/endpoints.test.js

```diff
@@ -0,0 +1,36 @@
+const db = require('../db/connection');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const request = require('supertest');
+const testData = require('../db/data/test-data/index');
+const { toBeSortedBy } = require('jest-sorted');
+
+
+afterAll(() => {
+    db.end;
+})
+
+beforeEach(()=>{
+    return seed(testData);
+})
+
+describe('GET /api/topics', () => {
+    test('testing that it returns a 200 status code ', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    });
+    test('responds with topics with following keys', () => {
+        return request(app)
+        .get('/api/topics')
+        .then((response) => {
+            const topics  = response.body.topics;
+            topics.forEach((topic) => {
+                expect(Object.keys(topic)).toHaveLength(2)
+                expect(topic).toHaveProperty("slug")
+                expect(topic).toHaveProperty("description")
+                
```

Comment:
Nice 👍 You could also test that the property values are strings.

Also think about what generic error we could test for here. E.g. what if someone misspells the endpoint.